pipeline/alert.go

```diff
@@ -1254,3 +1258,75 @@ func (a *AlertNode) Talk() *TalkHandler {
 type TalkHandler struct {
 	*AlertNode
 }
+
+// Send the alert using SNMP traps.
+// To allow Kapacitor to post SNMP traps,
+//
+// Example:
+//    [snmptrap]
+//      enabled = true
+//      target-ip = "127.0.0.1"
+//      target-port = 9162
+//      community = "public"
+//      version = "2c"
+//      state-changes-only = false
+//
+// In order to not post a message every alert interval
+// use AlertNode.StateChangesOnly so that only events
+// where the alert changed state are posted to the channel.
+//
+// Example:
+//    stream
+//         |alert()
+//             .snmpTrap()
+//             .data('1.3.6.1.2.1.1.7', 'i', '{{ index .Field "value" }}')
+//
+// Send alerts to `target-ip:target-port` on OID '1.3.6.1.2.1.1.7'
+//
+// tick:property
+func (a *AlertNode) SnmpTrap(trapOid string, dataList ...[]interface{}) *SnmpTrapHandler {
+	snmpTrap := &SnmpTrapHandler{
+		AlertNode: a,
+		TrapOid:   trapOid,
+		DataList:  dataList,
+	}
+	a.SnmpTrapHandlers = append(a.SnmpTrapHandlers, snmpTrap)
+	return snmpTrap
+}
+
+// SNMPTrap AlertHandler
+// tick:embedded:AlertNode.SnmpTrap
+type SnmpTrapHandler struct {
+	*AlertNode
+
+	TrapOid string
+	// List of trap data.
+	// tick:ignore
+	DataList [][]interface{} `tick:"Data"`
```

Comment:
Hello ! in fact, I'm just trying to get [][]string but I don't know how to get that 
from `alert.go` line 250
```
                tpattrtmpl, err := text.New("trap").Parse(attr.(string))
```
How can I get string from this stuff ?